impl/src/test/java/org/ehcache/GettingStarted.java

```diff
@@ -340,6 +341,24 @@ public void writeBehindCache() throws ClassNotFoundException {
     // end::writeBehindCache[]  
   }
 
+  @Test
+  public void testUpdateResourcesInRuntime() {
+    CacheConfiguration<Long, String> tieredCacheConfiguration = CacheConfigurationBuilder.newCacheConfigurationBuilder()
+        .withResourcePools(ResourcePoolsBuilder.newResourcePoolsBuilder()
+            .heap(10, EntryUnit.ENTRIES)
+            .disk(10, EntryUnit.ENTRIES).build())
+        .buildConfig(Long.class, String.class);
+    CacheManager cacheManager = CacheManagerBuilder.newCacheManagerBuilder().withCache("tieredCache", tieredCacheConfiguration)
+        .with(new PersistenceConfiguration(new File(System.getProperty("java.io.tmpdir") + "/tiered-cache-data")))
+        .build(true);
+
+    Cache<Long, String> tieredCache = cacheManager.getCache("tieredCache", Long.class, String.class);
+    ResourcePools pools = ResourcePoolsBuilder.newResourcePoolsBuilder().heap(20L, EntryUnit.ENTRIES).disk(2L, EntryUnit.ENTRIES).build();
+    tieredCache.getRuntimeConfiguration().updateResourcePools(pools);
```

Comment:
From what I see in the code, this call will have no effects.
Doing `tieredCache.getRuntimeConfiguration().getResourcePools().getPoolForResource(ResourceType.Core.HEAP).getSize()` after it will still return `10`.